### PR TITLE
feat(sdk): OIDC4VCI_ID1

### PR DESCRIPTION
**Pull request by mishasizov-SK**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,6 +12,7 @@ else
 endif
 
 ALPINE_VER ?= 3.18
+GO_ALPINE_VER ?= 3.18
```
Comment (rolsonquadras) [inline Makefile:15]: Looks like this variable is unused.

File: cmd/wallet-sdk-gomobile/openid4ci/supportedcredential.go [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-
-package openid4ci
-
-import (
-	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/api"
-	"github.com/trustbloc/wallet-sdk/pkg/models/issuer"
-)
-
-// SupportedCredentials represents the credentials (types and formats) that an issuer can issue.
-type SupportedCredentials struct {
```
Comment (rolsonquadras) [inline cmd/wallet-sdk-gomobile/openid4ci/supportedcredential.go:15]: Any reason to move the struct and methods to new file ?

**Review by rolsonquadras — COMMENTED** [review]
Couple of minor comments. 

Also, marked the PR as WIP to avoid merging before testing the flows manually.

File: Makefile (hunk 1 above)
Comment (mishasizov-SK) [reply Makefile:15]: I see it is used in [Makefile](https://github.com/trustbloc/wallet-sdk/blob/main/Makefile#L82) so decided to add.

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**